### PR TITLE
Completing OSSR onboarding: adding codemeta.json (and .zenodo.json) 

### PR DESCRIPTION
**Pull request by cosimoNigro**
This PR adds a `codemeta.json` file listing some metadata that should be automatically parsed by the OSSR when looking for software. I generated it via https://codemeta.github.io/codemeta-generator/.

@javierrico, if you want I can add a similar file for zenodo. 
I guess that now, at each version you release on GitHub, you are filling manually the description [in the zenodo page](https://zenodo.org/record/4601451), aren't you?
Otherwise I think the text in the GitHub release is copied and displayed by zenodo, right?

A `.zenodo.json` file can be created that is automatically used by zenodo to fill the details in the release page.
See a similar example in agnpy
https://github.com/cosimoNigro/agnpy/blob/master/.zenodo.json
and the resulting output
https://zenodo.org/record/5748085

Not sure if markdown syntax (e.g. links, boldface) can be parsed from the .zenodo.json file, though.

Let me know if you want me to add also the `.zenodo.json`.

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "macOs"
+    ],
+    "softwareRequirements": [
+        "ROOT"
+    ],
+    "author": [
+        {
+            "@type": "Person",
+            "@id": "https://orcid.org/0000-0003-4137-1134",
+            "givenName": "Javi",
```
Comment (javierrico) [inline codemeta.json]: My name is Javier

File: codemeta.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/GPL-3.0",
+    "codeRepository": "https://github.com/javierrico/gLike",
+    "dateCreated": "2018-11-13",
+    "datePublished": "2020-07-30",
+    "dateModified": "2021-03-12",
```
Comment (javierrico) [inline codemeta.json:8]: What do these dates refer to?

File: codemeta.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    "downloadUrl": "https://github.com/javierrico/gLike/releases/tag/v00.09.03",
```
Comment (javierrico) [inline codemeta.json:9]: Does this mean that we need to update this file for every release?

File: codemeta.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    "issueTracker": "https://github.com/javierrico/gLike/issues",
+    "name": "gLike",
+    "version": "v00.09.03",
```
Comment (javierrico) [inline codemeta.json:12]: Also needs to be updated right?

File: codemeta.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    "identifier": "10.5281/zenodo.3967385",
+    "description": "gLike is a general-purpose ROOT-based code framework for the numerical maximization of joint likelihood functions. The joint likelihood function has one free parameter (named g) and as many nuisance parameters as wanted, which will be profiled in the maximization process.",
```
Comment (javierrico) [inline codemeta.json]: Is it possible to add the current description at the Zenodo page which is a bit longer than this one?

**Review by javierrico — COMMENTED** [review]
Hi @cosimoNigro, yes this is great, as you were guessing I had to change this manually for every release, which was a bit of a pain, now it'll be much better
Thanks, just a few minor comments to consider and I will merge

File: codemeta.json (hunk 2 above)
Comment (cosimoNigro) [reply codemeta.json:8]: Sorry, used your full name

File: codemeta.json (hunk 3 above)
Comment (cosimoNigro) [reply codemeta.json:9]: yes, you should update each time with the date and version of the release manually.
This correspond to the "Current version of the software" in the [codemeta generator](https://codemeta.github.io/codemeta-generator/).

File: codemeta.json (hunk 4 above)
Comment (cosimoNigro) [reply codemeta.json:12]: see previous comment

**Comment by cosimoNigro** [comment]
@javierrico I also added a .zenodo.json, after merging you can make a dummy release from github (e.g. `v00.09.01test`) just to test that it's correctly parsed by zenodo.

For what concerns the description, you asked about it in the `codemeta.json`, the problem is that `.json` format does not support multi-line strings. I can make newlines with the `\n` but would not know how to render the itemise, nor if the zenodo API will parse it correctly, I doubt it.

Since the itemise lists only examples and is available in the main github page, can we be happy just with the introductory lines?
If you check I have also [shortened the agnpy description](https://zenodo.org/record/5748085). If people want more details they can go to the github with a click, no?

File: codemeta.json (hunk 1 above)
Comment (cosimoNigro) [reply codemeta.json]: Sorry, used your full name

File: codemeta.json (hunk 2 above)
Comment (cosimoNigro) [reply codemeta.json:8]: They are the creation and first release date, see "The software itself" section of the [codemeta generator](https://codemeta.github.io/codemeta-generator/). 
I obtained the creation date, `dateCreated`, from the GitHub repos API (just goggle https://api.github.com/repos/javierrico/glike).
And the date of the first release `datePublished`, simply going through your releases in GitHub.

**Comment by cosimoNigro** [comment]
The .zenodo.json was generated from the codemeta.json, in order to comply with the [OSSR directives](https://escape2020.pages.in2p3.fr/wp3/ossr-pages/page/contribute/checklist/).

**Comment by javierrico** [comment]
Thanks @cosimoNigro, all clear

**Pull request merged**